app/src/main/java/io/spaceapi/community/myhackerspace/Main.java

```diff
@@ -676,7 +698,7 @@ private void populateDataHs() {
                     .setCompoundDrawablesWithIntrinsicBounds(
                         android.R.drawable.presence_busy, 0, 0, 0);
             }
-            if (data.state.message != null) {
+            if (data.state != null && data.state.message != null) {
```

Comment:
Isn't there a safe call operator in kotlin?
```suggestion
            if (data.state?.message != null) {
```

Reply:
This is Java, not Kotlin.

Reply:
I just assumed the app was written in Kotlin and not Java, didn't even look at the file ending :smile: